plugins/modules/clusters.py

```diff
@@ -68,6 +68,12 @@
 
 import os
 import traceback
+
+try:
```

Comment:
I know we do this for sake of the CI, but looks strange to me, we should create an issue to figure out what's going on. Not a blocker for this change, let's go with it.

Reply:
@tonyskapunk FYI Issue created at (https://github.com/vjayaramrh/assistedinstaller/issues/81)
cc: @agonzalezrh